app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
```

Comment:
- [ ] A user model needs to have associations for likes and comments as well. Please add them like this 
 ```
 has_many :comments, foreign_key: 'author_id'
 has_many :likes, foreign_key: 'author_id'
 ```

Reply:
correction done